React/Fabric/Mounting/ComponentViews/ScrollView/RCTScrollViewComponentView.mm

```diff
@@ -428,8 +455,11 @@ - (void)scrollViewDidScroll:(RCTUIScrollView *)scrollView // TODO(macOS GH#774)
     if (_eventEmitter) {
       std::static_pointer_cast<ScrollViewEventEmitter const>(_eventEmitter)->onScroll([self _scrollViewMetrics]);
     }
-
+    // Once Fabric implements proper NativeAnimationDriver, this should be removed.
+    // This is just a workaround to allow animations based on onScroll event.
+#if !TARGET_OS_OSX // TODO(macOS GH#774)
```

Comment:
Is this comment something you wrote? Why is it showing up in this PR?

Reply:
Ah, this comment actually belongs here:

https://github.com/facebook/react-native/blob/main/React/Fabric/Mounting/ComponentViews/ScrollView/RCTScrollViewComponentView.mm#L54-L58

It was copy pasta'd to the call site. Will remove